intelmq/tests/lib/test_splitreports.py

```diff
@@ -0,0 +1,158 @@
+"""
```

Comment:
change accordingly (remove test_splitreports.py) to the code that was merged in CollectorBot (https://github.com/certtools/intelmq/blob/master/intelmq/lib/bot.py#L576)